cmd/meroxa/turbine/utils.go

```diff
@@ -485,9 +485,12 @@ func uploadFile(ctx context.Context, logger log.Logger, filePath, url string) er
 	if res != nil && res.Body != nil {
 		defer res.Body.Close()
 	}
-	if err != nil {
+	if clientErr != nil || (res.StatusCode < http.StatusOK || res.StatusCode >= http.StatusNoContent) {
```

Comment:
My understanding is that this is checking for 2XX HTTP status responses - is it an issue if there is a `204 StatusNoContent` returned here? Thoughts on just allowing all 2xx responses?

Reply:
i felt like NoContent would not be a good time if we were uploading something of non-zero size.  I'm not an expert in HTTP responses.

Reply:
if it makes sense, i can change this to be any 200s response